arch/arm/mach-imx/imx8m/Kconfig

```diff
@@ -2,8 +2,10 @@ if ARCH_IMX8M
 
 config IMX8M
 	bool
-	select HAS_CAAM
 	select ROM_UNIFIED_SECTIONS
+	help
+	  This architecture has CAAM but we have decided to unselect
+	  it so we can enable Foundries.io secure boot (spl + itb)
```

Comment:
Can you drop this extra help change? While useful, it can easily generate conflicts on rebases (just make it part of the commit message instead).